main.py

```diff
@@ -7,6 +7,22 @@
 import discord
 from discord_slash import SlashCommand, SlashContext
 import os
+from dotenv import load_dotenv
+
+# loads the dotenv variables into the environment
+load_dotenv()
+
+# if its not present in the .env then it will load it from the environ, if not present it will be set to None
+SSLString = '?sslmode=require' if os.getenv(
+    'SSL_MODE') == None or os.getenv('SSL_MODE') == 'True' else ''
+
+if (os.getenv('DATABASE_URL') == None):
+    POSTGRESS_USER = os.getenv('POSTGRESS_USER')
+    DB_PASSWORD = os.getenv('DB_PASSWORD')
+    DB_NAME = os.getenv('DB_NAME')
+    os.environ['DATABASE_URL'] = f"postgres://{POSTGRESS_USER}:{DB_PASSWORD}@localhost:5432/{DB_NAME}{SSLString}"
```

Comment:
```suggestion
    DB_USER = os.getenv('DB_USER')
    DB_PASSWORD = os.getenv('DB_PASSWORD')
    DB_NAME = os.getenv('DB_NAME')
    os.environ['DATABASE_URL'] = f"postgres://{DB_USER}:{DB_PASSWORD}@localhost:5432/{DB_NAME}{SSLString}"
```

Reply:
Maybe it would be best to rename this environment variable to `DB_USER` to keep it consistent with the other ones